kubernetes/helm_charts/core/uci/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: uciapi
```

Comment:
Why dont we use the name as `uci`. And then in confirmap references, we can use `{{ .Chart.Name }}-config`

Reply:
Changed the name to uci.